seal_test.go

```diff
@@ -54,26 +54,33 @@ func TestSealKeyToTPM(t *testing.T) {
 	key := make([]byte, 64)
 	rand.Read(key)
 
-	run := func(t *testing.T, tpm *TPMConnection, params *KeyCreationParams) (authKeyBytes []byte) {
+	runNoCleanup := func(t *testing.T, tpm *TPMConnection, params *KeyCreationParams) (authKeyBytes []byte, path string, cleanup func()) {
```

Comment:
I think runDeferredCleanup might be a slightly clearer name, given that this returns a cleanup after all